### PR TITLE
.NET 5 updates

### LINKED ISSUE #21868: Update for .NET 5
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7d06515c-6936-7410-02aa-bdabb6b6b8d0
* Version Independent ID: a20cf160-8040-47a4-6996-30a0facf2506
* Content: [Develop libraries with the .NET Core CLI - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/tutorials/libraries)
* Content Source: [docs/core/tutorials/libraries.md](https://github.com/dotnet/docs/blob/master/docs/core/tutorials/libraries.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @cartermp
* Microsoft Alias: **tdykstra**

### LINKED ISSUE #21869: Update for .NET 5
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f9b54562-9072-a864-be3d-44ed7965c798
* Version Independent ID: a69ec985-ed05-90dd-a686-dc956b0cf5df
* Content: [Organizing and testing projects with the .NET Core CLI - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/tutorials/testing-with-cli)
* Content Source: [docs/core/tutorials/testing-with-cli.md](https://github.com/dotnet/docs/blob/master/docs/core/tutorials/testing-with-cli.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @cartermp
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #21869
Fixes #21868

### CODE REVIEW

File: docs/core/tutorials/libraries.md [hunk 1]
```diff
@@ -27,35 +27,27 @@ Additionally, if you wish to support older .NET Framework targets, you need to i
 | 4.0                    | Windows SDK for Windows 7 and .NET Framework 4         |
 | 2.0, 3.0, and 3.5      | .NET Framework 3.5 SP1 Runtime (or Windows 8+ version) |
 
-## How to target .NET Standard
+## How to target .NET 5.0 or .NET Standard
 
-If you're not familiar with .NET Standard, refer to [.NET Standard](../../standard/net-standard.md) to learn more.
+For guidance on how to choose between targeting .NET 5.0 or .NET Standard see [.NET 5 and .NET Standard](../../standard/net-standard. You control your project's target framework by adding it to your project file (`.csproj` or `.fsproj`).
```
Comment (IEvangelist) [inline docs/core/tutorials/libraries.md:32]:
```suggestion
You control your project's target framework by adding it to your project file (*.csproj* or *.fsproj*). For guidance on how to choose between targeting .NET 5.0 or .NET Standard see [.NET 5 and .NET Standard](../../standard/net-standard).
```

- Restructure (and order) sentences
- Fix open-ended MD link
- Change from code to italic for file refs

**Review by IEvangelist — APPROVED** [review]
One minor suggestion, then :shipit:

**Comment by gewarren** [comment]
@tdykstra Looks like this introduced a warning.

**Comment by tdykstra** [comment]
@gewarren Yes, thanks, fixed in #21992

**Pull request merged**